src/pages/d/[id].tsx

```diff
@@ -26,7 +25,7 @@ export default function Home() {
 
     (async () => {
       logger.info?.(`Connecting to worker at ${workerWsURI}`);
-      const userID = nanoid();
+      const userID = "reflect-user";
```

Comment:
I understand taht we want multiple windows to share the userID but it seems a little bit "ugly" to have everyone use the same user ID. I know it does not really matter but maybe we should not call this userID any more?